tests/test_needle_plot.py

```diff
@@ -0,0 +1,223 @@
+import os
+from tests.dashbio_demos.app_needle_plot import load_mutation_data, DATAPATH, DEMO_DATA
+from .test_common_features import init_demo_app, template_test_component_single_prop, \
+    PROP_TYPES, COMPONENT_REACT_BASE, generate_subprop_test
+
+APP_NAME = os.path.basename(__file__).replace('test_', '').replace('.py', '').replace('_', '-')
+
+TEST_DATA = load_mutation_data('{}{}'.format(DATAPATH, DEMO_DATA[0]['mutData']))
+
+
+# Demo app tests
+@init_demo_app(APP_NAME)
+def test_click_app_link_from_gallery(dash_threaded, selenium):
+
+    assert selenium.current_url.replace('http://localhost:8050', '').strip('/') == \
+           'dash-bio/{}'.format(APP_NAME)
+
+
+# Needle Plot component tests
```

Comment:
```suggestion
# Component tests
```
Just so it's the same as the boilerplate, plus we already know it's NeedlePlot (filename).